frb_example/with_flutter/rust/src/bridge_generated.io.rs

```diff
@@ -197,6 +197,15 @@ impl Wire2Api<Vec<u8>> for *mut wire_uint_8_list {
         }
     }
 }
+// Section: dart opaque related functions
+
+#[no_mangle]
+pub extern "C" fn drop_DartOpaqueFlutterRustBridgeExample(ptr: usize) {
```

Comment:
same as before - is it possible we put it to frb_dart

Reply:
fixed